src/core/db/schema.ts

```diff
@@ -0,0 +1,38 @@
+import { integer, sqliteTable, text } from "drizzle-orm/sqlite-core";
+import { INTENT_STATUS } from "../../types";
+
+export const projects = sqliteTable("projects", {
+  id: text("id").primaryKey(),
+  repoPath: text("repo_path").notNull().unique(),
+  repoName: text("repo_name").notNull(),
+});
+
+export const branches = sqliteTable("branches", {
+  id: text("id").primaryKey(),
+  projectId: text("project_id")
+    .notNull()
+    .references(() => projects.id),
+  name: text("name").notNull(),
+});
```

Comment:
Would it be better to add a unique constraint to prevent a project from having two branches with the same name? It would look something like the following:

```ts
// might not be the correct drizzle syntax
(table) => [
  unique().on(table.projectId, table.name),
]
```

What do you think?